app/Http/Controllers/AuthController.php

```diff
@@ -41,13 +40,13 @@ public function install(Request $request)
             // Response contains access_token, context, userId, userEmail
             $data = json_decode($response->getBody(), true);
             if ($statusCode == 200) {
-                $store = Store::where('domain', $data['context'])->first();
+                $store = Store::where('context', $data['context'])->first();
```

Comment:
How did this even work before?

Reply:
In staging it's breaking currently, I need to refresh the db and re-run migrations.

![Screen Shot 2021-02-03 at 13 50 28](https://user-images.githubusercontent.com/511559/106749444-cb44b480-6626-11eb-8a07-7610f2ba91a5.png)

Reply:
Works again, had to execute `php artisan migrate:refresh` for renewing migrations.

Reply:
Running `php artisan migrate:refresh` on production, will cause loss of all stored data. Does this mean that there is no migration?

Reply:
Since there is no production yet, it doesn't matter. I've updated the create table statements of existing migration.

Reply:
Fine then.